Makefile

```diff
@@ -36,9 +36,9 @@ test-unit:
 test-e2e:
 	./test/e2e-tests.sh
 
-# Run E2E tests from the current repo for serving+eventing+knativeKafka
+# Run the Kafka operator e2e hook
```

Comment:
The old comment was more correct. It's actually running everything and includes Kafka.